src/components/ui/Multivalue/MultivalueTag.tsx

```diff
@@ -61,17 +69,37 @@ const MultivalueTag: React.FunctionComponent<MultivalueTagProps> = (props) => {
                     </Tag>
                 })}
             </div>
-            <div
-                className={cn(
-                    styles.iconContainer,
-                    {[styles.disabled]: props.disabled}
-                )}
-                onClick={handleOpen}
-            >
-                <Icon type="folder-open"/>
-            </div>
+            {props.popupRowMetaDone || !props.popupBcName
```

Comment:
```tsx
const loading = props.popupRowMetaDone || !props.popupBcName
// ...
<div className={cn(styles.iconContainer, { [styles.disabled]: props.disabled })}>
    <Icon
        type={loading ? 'folder-open' : 'loading'}
        spin={loading}
        onClick={loading ? handleOpen : undefined}
    />
</div>
```

Reply:
fixed with inverted logic of prop `loading`